doc/before-after-reboot-checks.md

```diff
@@ -0,0 +1,48 @@
+# Before and After Reboot Checks
+
+CLUO provides optional functionality for running arbitrary actions before and
+after the rebooting of a node using node labels and annotations.
+
+## Configuring the `update-operator`
+
+When depolying the `update-operator`, you can provide a comma-separated list of
+node annotations to look for before a reboot should occur, and another one for
+after a reboot has occured. The list is provided as a command line argument to
+the binary -
+
+```bash
+update-operator --before-reboot-annotations anno1,anno2 --after-reboot-annotations anno3,anno4
+```
+
+## Making a check
```

Comment:
I was trying to avoid referring to them as pods here, since they really don't have to be. It could be a cronjob on someones machine checking the labels every minute and then running `kubectl annotate`. Not that that's the recommended way to do things (hence the examples being pods) but I think there is value in keeping it flexible.